src/proj.h

```diff
@@ -341,6 +341,9 @@ void proj_errno_restore (PJ *P, int err);
 
 char *proj_get_definition (PJ *P);
 
+/* deallocate anything obtained from PROJ.4 */
```

Comment:
Yes - will make that clear when getting around to writing the formal documentation, and update the comment for clarity